db/r2decompile

```diff
@@ -0,0 +1,31 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/securisec/r2decompile"
+R2PM_DESC "[r2decompile] Use local opensource retdec to decompile functions"
+
+R2PM_DOC="
+usage: $r2decompile [-h] [-p] [-t FILE]
+
+r2decompile help
+
+Optional arguments:
+  -h, --help  Show this help message and exit.
+  -t TMP      Set temp file for decompiled code
+  -p          Print dicompilation to stdout
+
+Invoke from inside r2 shell with $dec
+"
+
+R2PM_INSTALL() {
+    cd ${R2PM_GITDIR}r2decompile
+    npm install
+    cd -
+	echo '$'dec="#"'!'"pipe node ${R2PM_GITDIR}r2decompile/r2decompile.js" >> ~/.radare2rc || exit 1
```

Comment:
Use bindir from r2pm to honor -g and homedir properly

Reply:
Ok thats just an echo. You can put that script in bindir from r2pm and make jt accessible from r2 without having to set the full path

Reply:
not sure what you mean. all my other plugins follow this same format in r2pm